### PR TITLE
Settlement endpoint

### PR DESCRIPTION
**Pull request by kingel**

### CODE REVIEW

File: mollie/api/resources/settlement_refunds.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from .refunds import Refunds
+
+
+class SettlementRefunds(Refunds):
+    settlementt_id = None
```
Comment (whyscream) [inline mollie/api/resources/settlement_refunds.py]: Typo in variable name

File: tests/test_settlements.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+from mollie.api.objects.settlement import Settlement
+from mollie.api.resources.settlements import Settlements
+from tests.utils import assert_list_object
+
+SETTLEMENT_ID = 'stl_jDk30akdN'
+
+
+def test_settlements_resource_class(client, response):
+    response.get('https://api.mollie.com/v2/settlements', 'settlements_list')
+    client.settlements.list()
```
Comment (whyscream) [inline tests/test_settlements.py]: Why are you performing an API call here? The test also succeeds when you don't perform the call.

File: tests/test_settlements.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        (None, IdentifierError),
+        ("foo", IdentifierError),
+        ("next", None),  # Valid
+        ("open", None),  # Valid
+        ("stl_", None),  # Valid
```
Comment (whyscream) [inline tests/test_settlements.py]: Actually, this is not a valid example of a settlement ID. How about using the `SETTLEMENT_ID` constant you already have?

File: tests/test_settlements.py [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+from tests.utils import assert_list_object
+
+SETTLEMENT_ID = 'stl_jDk30akdN'
+
+
+def test_settlements_resource_class(client, response):
+    response.get('https://api.mollie.com/v2/settlements', 'settlements_list')
+    client.settlements.list()
+
+    assert isinstance(Settlement.get_resource_class(client), Settlements)
```
Comment (whyscream) [inline tests/test_settlements.py]: Normally, the `get_resource_class()` method is used when the list response is paginated, and you call `get_next()` or `get_previous()`. If your goal for this test is coverage for the `get_resource_class` method, that's the way to go.

File: tests/test_settlements.py (hunk 2 above)
Comment (kingel) [reply tests/test_settlements.py]: copy leftovers i think

File: mollie/api/resources/settlement_refunds.py (hunk 1 above)
Comment (kingel) [reply mollie/api/resources/settlement_refunds.py]: solved

File: tests/test_settlements.py (hunk 3 above)
Comment (kingel) [reply tests/test_settlements.py]: done

File: tests/test_settlements.py (hunk 4 above)
Comment (kingel) [reply tests/test_settlements.py]: removed it completely

**Comment by kingel** [comment]
> There are 2 tests in `test_payments.py` current marked as expected failure due to settlements not being implemented. Could you address those too?

Solved

**Pull request merged**